internal/patches.go

```diff
@@ -4,20 +4,39 @@ package internal
 import (
 	"errors"
 	"fmt"
+	"os"
 	"path"
+	"path/filepath"
 
 	yaml "gopkg.in/yaml.v3"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/kustomize/api/krusty"
 	"sigs.k8s.io/kustomize/kyaml/filesys"
+
+	"open-cluster-management.io/policy-generator-plugin/internal/types"
+)
+
+const (
+	localSchemaFileName = "schema.json"
+	kustomizeDir        = "kustomize"
 )
 
+type KustomizeJSON struct {
+	types.Filepath `json:"openapi,omitempty" yaml:"openapi,omitempty"`
```

Comment:
This is less confusing
```suggestion
	openapi types.Filepath `json:"openapi,omitempty" yaml:"openapi,omitempty"`
```

Reply:
Fixed. Renamed to OpenAPI